Formplayer/settings-sample.cfg

```diff
@@ -0,0 +1,7 @@
+[default]
+url = https://staging.commcarehq.org/
+login_username = automation.user.commcarehq@gmail.com
+login_password = pass@123
+staging_auth_key = "DRCAKX6AG3V3Q7JWV27KIPOWVMAYJ3R4"
+prod_auth_key = "YRJSX3BARTDVYUUQKIQUX3EIKQUZZS5P"
+
```

Comment:
even in the sample file, do not provide password and keys.
Keep the same template as before, just remove the elements which are not required. In the sample file you can provide the URL value but not the others.

Reply:
Is there a way so that this commit does not appear in the commit history? Otherwise we need to change the password,